### ISSUE TITLE
Backward incompatibility of dashboards from `v5` to `v4`

### ISSUE THREAD

**Issue by FUSAKLA** [grafana/grafana#10624]
- What Grafana version are you using?
`Version 5.0.0-pre1 (commit: 77cc546)`
and
`Version 4.6.3 (commit: 7a06a47)`

- What OS are you running grafana on?
official upstream docker image in kubernetes
using MySQL as backend

- What did you do?
Upgraded form `4.6.2` to `5.0.0-pre1` and than downgraded back to `4.6.3`
 
- What was the expected result?
to see all dashboard as they were (with minor differences) as the changelog stated 

- What happened instead?
grafana starts ok, i can log in and all dashboard that i did not modify in the `5` version are ok
but those i edited and saved won't display properly in the `4.6.3`.
exactly panels are not shown at all (template variables, header and so no is ok but panels not even rows are displayed)

changelog states that:
```
 The change is backwards compatible. Grafana will automatically upgrade your dashboards to the new schema and position panels to match your existing layout. There might be minor differences in panel height.
```
 
dashboard in version `v5.0.0-pre1`
![screenshot from 2018-01-26 02-09-47](https://user-images.githubusercontent.com/6112562/35420501-1782f06e-023e-11e8-9237-e620db936959.png)

the same dashboard after downgrade to `4.6.3`
![screenshot from 2018-01-26 02-06-27](https://user-images.githubusercontent.com/6112562/35420508-2bc2e98a-023e-11e8-950a-982a36e938fb.png)

**Comment by torkelo** [comment]
Yes, dashboards created or exported from v5 are not compatible with v4 due to a new panel positioning system, https://github.com/grafana/grafana/blob/master/CHANGELOG.md#new-dashboard-grid

**Comment by bergquist** [comment]
I think we should be more explicit about what we mean with backward compatible. https://en.wikipedia.org/wiki/Backward_compatibility

You can use dashboard history to restore previous versions of the dashboard that will work in 4.6.3.
http://docs.grafana.org/reference/dashboard_history/

**Comment by FUSAKLA** [comment]
but the changelog states it's backward compatible
there is no mention about this incompatibility 

Also those were not created or exported. Those were created in `v4` than I upgraded grafana to `v5` after that it displayed dashboards ok but as I edited them and saved it clearly saved it in backward incompatible format because after the downgrade they are not shown correctly.

@bergquist 
great i didn't thought of that! thanks :)

And I agree the changelog could be more specific on this.
